tests/api2/test_032_ad_kerberos.py

```diff
@@ -8,27 +8,24 @@
 
 apifolder = os.getcwd()
 sys.path.append(apifolder)
-# from functions import make_ws_request
-from functions import PUT, POST, GET, DELETE, SSH_TEST, wait_on_job
+from functions import SSH_TEST
```

Comment:
imports that can be removed:
`time`
`calendar.timegm`
`pytest_dependency.depends`
`config.ADNameServer`